oneliners.cc

```diff
@@ -0,0 +1,28 @@
+#include <algorithm>
+#include <iostream>
+#include <numeric>
+#include <vector>
+#include <sstream>
+
+using namespace std;
+
+int main()
+{
+    vector<int> v(100);
+    iota(v.begin(), v.end(), 0);
```

Comment:
But it's 2 lines (if I got it right, and it should be solution for (1) ).

Reply:
Hmm, right. Is something like 
vector<int> v{1,3,5,7,45} is an answer? or should we do something that can be extended to long vectors?

Reply:
Well, the assignment does not say you have push _different_ elements ;-)

Reply:
Yes, but if I should use it as a base for all the other operations, it will be a bad test case to have like
vector v{100, 5}...
Of course I could do it and then create a more interesting one, to test the other operations :)